demos/pedestrian_tracker_demo/src/cnn.cpp

```diff
@@ -77,7 +77,9 @@ void CnnBase::InferBatch(
             matU8ToBlob<uint8_t>(frames[batch_i + b], input_blob_, b);
         }
 
-        infer_request_.SetBatch(current_batch_size);
+        if (ie_.GetConfig(deviceName_, CONFIG_KEY(DYN_BATCH_ENABLED)).as<std::string>() == PluginConfigParams::YES) {
```

Comment:
```suggestion
        if (1 != batch_size) {
```
`GetConfig` will throw exceptions, so it doesn't really differ from calling `infer_request_.SetBatch(current_batch_size);` unconditionally and catching exceptions. But exception catch can lead to a performance degradation. That's why I suggest ` if (1 != batch_size)`